lib/manager/pipenv/artifacts.ts

```diff
@@ -28,14 +48,15 @@ export async function updateArtifacts({
     const localPipfileFileName = join(config.localDir, pipfileName);
     await outputFile(localPipfileFileName, newPipfileContent);
     const localLockFileName = join(config.localDir, lockFileName);
-    const cmd = 'pipenv lock';
+    const cmd = ['pip install pipenv', 'pipenv lock'];
```

Comment:
Does pipenv itself leave a trace of what version it uses?

BTW I also now recall that some people run `pipenv` off the github repo because it was so long since the previous release.

Reply:
I think pip install latest version, we need to check, maybe there are some pipenv constaints in pipfile (at least if user defined)

Reply:
Yes, we need to check if the Pipfile or Pipfile.lock lists an exact `pipenv` version to use.